actions/port-forward/action.yaml

```diff
@@ -24,15 +28,16 @@ runs:
         version: ${{ inputs.kubectl-version }}
 
     - name: Establish port forward
-      shell: bash
+      env:
+        TIMEOUT: "${{ inputs.timeout }}"
```

Comment:
Is a remap required here?

Reply:
it just makes it easier to use in the bash code.